index.bs

```diff
@@ -12839,6 +12871,34 @@ running the algorithm for an {{AudioNode}}, using an <a>input
 buffer</a> and the value(s) of the {{AudioParam}}(s) of this
 {{AudioNode}} as the input for this algorithm.
 
+
+<h3 id="error-handling-on-a-running-audio-context">
+    Error Handling for System Audio Resources on a running AudioContext</h3>
+
+In the event of an audio system resource error, the <a>rendering thread</a>
+performs the following steps upon receiving the error:
+
+1. Set {{[[rendering thread state]]}} of the {{AudioContext}} to
+    "{{AudioContextState/suspended}}".
+
+1. <a href="https://html.spec.whatwg.org/multipage/media.html#queue-a-media-element-task">
+    Queue a media element task</a> to execute the following steps:
+
+    1. <a spec="dom" lt="fire an event">fire an event</a> named
+        {{AudioContext/error}} at the associated {{AudioContext}}.
```

Comment:
onerror?

Reply:
The event is called `"error"`. This in turns call `onerror` on the context via normal even `EventTarget` rules.

Reply:
@gsinafirooz If the comment/question has more detailed, it would be more helpful to others.